Clipy/Sources/Services/PasteService.swift

```diff
@@ -140,20 +141,32 @@ extension PasteService {
 extension PasteService {
     func paste() {
         if !AppEnvironment.current.defaults.bool(forKey: Constants.UserDefaults.inputPasteCommand) { return }
+        let keyCode = findKeyCode(character: "V")
+        let scanCode = CGKeyCode(keyCode > 0 ? keyCode : 9)
 
         DispatchQueue.main.async {
             let source = CGEventSource(stateID: .combinedSessionState)
             // Disable local keyboard events while pasting
             source?.setLocalEventsFilterDuringSuppressionState([.permitLocalMouseEvents, .permitSystemDefinedEvents], state: .eventSuppressionStateSuppressionInterval)
             // Press Command + V
-            let keyVDown = CGEvent(keyboardEventSource: source, virtualKey: CGKeyCode(9), keyDown: true)
+            let keyVDown = CGEvent(keyboardEventSource: source, virtualKey: scanCode, keyDown: true)
             keyVDown?.flags = .maskCommand
             // Release Command + V
-            let keyVUp = CGEvent(keyboardEventSource: source, virtualKey: CGKeyCode(9), keyDown: false)
+            let keyVUp = CGEvent(keyboardEventSource: source, virtualKey: scanCode, keyDown: false)
             keyVUp?.flags = .maskCommand
             // Post Paste Command
             keyVDown?.post(tap: .cgAnnotatedSessionEventTap)
             keyVUp?.post(tap: .cgAnnotatedSessionEventTap)
         }
     }
+
+    func findKeyCode(character: String) -> Int {
```

Comment:
@jcw- Thank you very good PR 👍  

I do not want to do searching for key codes every paste. Since the layout of the keyboard is not frequently changed, I would like to search only once at initialization as follows, what do you think?

```
private var pasteKeyCode: CGKeyCode = {
    for keyCode in 0...128 {
        let keyCombo = KeyCombo(keyCode: keyCode, carbonModifiers: 0)
        if keyCombo?.characters == "V" {
            return CGKeyCode(keyCode)
        }
    }
    return CGKeyCode(9)
}()
```

Reply:
@jcw- Thanks :) Can you add this change to the PR?

Reply:
@Econa77 You bet! Done and rebased